types/keys.go

```diff
@@ -12,4 +12,7 @@ const (
 var (
 	// CoinDecimals is the amount of staking tokens required for 1 unit
 	CoinDecimals = new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil)
+
+	// FeeToken is fee token symbol
+	FeeToken = "matic"
```

Comment:
May be it shouldn't sit here.  How about making it configurable in genesis file directly? Can be addressed in a separate PR though

Reply:
Agree. I will add it genesis as params. Which module should have it?

Reply:
It's so generic. I would suggest to keep it in auth/bank module. wdyt?

Reply:
Yeah we can have it in auth module. Can we add it in params even if we are using `cosmos/x/auth`?

Reply:
I think we cannot directly use `cosmos/x/auth`, it will fail in Params validation and Genesis Validation. I guess we can override them but that would be risky. Given this, `bank` looks a better option for our case.

Reply:
Let's put that in the next PR. We can review/merge this PR in that case.